uikit/buttonV2/Button.tsx

```diff
@@ -0,0 +1,53 @@
+import { FunctionComponent, ButtonHTMLAttributes } from "react";
+import ButtonPrimary from "./ButtonPrimary";
+
+type ButtonPrimaryProps = {
+    size?: "md" | "lg" | "xl";
+    type?: "fab" | "default";
+} & Omit<ButtonHTMLAttributes<HTMLButtonElement>, "size" | "type">;
+
+const buttonClasses = {
+    base: "flex flex-row items-center justify-center gap-2 active:scale-90 transition ease-out",
+    disabled: "bg-dark-neutral-primary opacity-40 cursor-not-allowed",
+    size: {
+        md: {
+            fab: "w-10 h-10 heading-h7",
+            default: "px-4 h-10 heading-h7",
+        },
+        lg: {
+            fab: "w-12 h-12 heading-h6",
+            default: "px-5 h-12 heading-h6",
+        },
+        xl: {
+            fab: "w-16 h-16 heading-h6",
+            default: "px-6 h-16 heading-h6",
+        },
+    },
+    type: {
+        fab: "rounded-lg",
+        default: "rounded-full",
+    },
+    variant: {
+        primary: "bg-dark-neutral-primary text-dark-background-default hover:bg-dark-neutral-soft",
+        secondary: "bg-light-neutral-subtle/10 py-3 text-dark-neutral-primary hover:bg-light-neutral-subtle/20",
```

Comment:
I think the `py-3` here is not necessary, because we already use fixed height

Reply:
I just copy-paste from your latest code haha
oke on it

Reply:
oops my bad, I think I missed that one haha

Reply:
updated ser